CONTRIBUTING.md

```diff
@@ -0,0 +1,13 @@
+Contributions are welcome!
+
+You can either pick one of the open issues and work on that, or provide enhacements according to your own needs.
+
+To contribute:
+1. Clone the repository
+1. Identify a modular contribution to the code (avoid to large contributions, since reviewing them will be complex)
+1. Create a branch in your repository where you tackle the "modular contributions" (this means that for multiple contributions tackling different functionalities, you should create different branches)
+1. When done, create a pull request against our repository
```

Comment:
I'd add:
- Include tests for you new code. (see setup_dev_env.sh and run.sh in the root to understand how tests can be run locally)
- Make sure tests are passing in CI before requesting review.